Content.Shared/Nutrition/Components/SatiationComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Content.Shared.Nutrition.EntitySystems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.Nutrition.Components;
+
+[Access(typeof(SatiationSystem))]
+[RegisterComponent, NetworkedComponent]
+[AutoGenerateComponentState, AutoGenerateComponentPause]
+public sealed partial class SatiationComponent : Component
+{
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
+    public Satiation Hunger = new();
+
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
+    public Satiation Thirst = new();
```

Comment:
make this store a collection of ProtoId<SatiationPrototype>

Reply:
Now Dictionary<ProtoId<SatiationTypePrototype>, Satiation>